person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  att_reader :id
+
+  attr_accessor :name, :age
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
- [ ] All method under the `private` declaration is treated as private method, but as demonstrated by the previous reviewer only the `of_age?` method should be private. That means the `can_use_services?` method should not be private. Kindly fix that.

  ```
  def can_use_services?
    of_age? || @parent_permission
  end

  private

  def of_age?
    @age >= 18
  end
  ```

Reply:
Well that is why I used explicit commit in my first commit as it is illustrated here in this article: [here](https://medium.com/@sologoubalex/private-methods-in-ruby-8a75b06b3b56) but nevertheless, this implicit method also seams to make sense, is just that one have to bear in mind the position of the private key word methods. what do you think?